linera-base/src/crypto/ed25519.rs

```diff
@@ -317,6 +352,8 @@ impl Ed25519Signature {
     }
 
     /// Verifies a batch of signatures.
+    // NOTE: This is unused now since we don't use ed25519 in consensus layer.
+    #[allow(unused)]
     pub fn verify_batch<'a, 'de, T, I>(value: &'a T, votes: I) -> Result<(), CryptoError>
```

Comment:
weird that Rust thinks this `pub` method is unused

Reply:
Would it make sense to use `#[expect(..)]`?

Reply:
It's not used anymore because we don't use Ed25519 batch verification anymore. I'm surprised it didn't complain about it earlier though.